code/modules/power/powernet.dm

```diff
@@ -331,6 +331,9 @@ var/global/powernets_broke = 0
 		if(H.gloves)
 			var/obj/item/clothing/gloves/G = H.gloves
 
+			if(G.name == "insulted gloves")
```

Comment:
Additionally, move this statement after the if(G.siemens_coefficient == 0) check below, in case they aren't actually shocked somehow. Imagine it insulting you but you're fine!